### PR TITLE
Rename virtual memory column from M_SIZE to M_VIRT

### PR DESCRIPTION
**Pull request by cgzones**
Closes: #325

### CODE REVIEW

**Review by BenBE — COMMENTED** [review]
LGTM. Maybe rename the `m_size` member in `struct Process_` too to have everything related to this column consistent?

**Comment by cgzones** [comment]
Also renamed the struct member

File: dragonflybsd/DragonFlyBSDProcessList.c [hunk 1]
```diff
@@ -442,7 +442,7 @@ void ProcessList_goThroughEntries(ProcessList* super, bool pauseProcessUpdate) {
          }
       }
 
-      proc->m_size = kproc->kp_vm_map_size / 1024 / pageSizeKb;
+      proc->m_virt = kproc->kp_vm_map_size / 1024 / pageSizeKb;
```
Comment (BenBE) [inline dragonflybsd/DragonFlyBSDProcessList.c:445]: Don't we have `pageSize` for this division instead?

File: freebsd/FreeBSDProcessList.c [hunk 2]
```diff
@@ -543,7 +543,7 @@ void ProcessList_goThroughEntries(ProcessList* super, bool pauseProcessUpdate) {
       }
 
       // from FreeBSD source /src/usr.bin/top/machine.c
-      proc->m_size = kproc->ki_size / 1024 / pageSizeKb;
+      proc->m_virt = kproc->ki_size / 1024 / pageSizeKb;
```
Comment (BenBE) [inline freebsd/FreeBSDProcessList.c:546]: Dito.

**Pull request merged**